pex/bin/pex.py

```diff
@@ -620,7 +625,13 @@ def make_relative_to_root(path):
 
 
 def main(args=None):
-  args = args or sys.argv[1:]
+  args = args[:] if args else sys.argv[1:]
+  # inherit-path used to be a boolean arg (so either was absent, or --inherit-path)
+  # Now it takes a string argument, so --inherit-path is invalid.
+  # Fix up the args we're about to parse to preserve backwards compatibility.
+  for i, arg in enumerate(args):
+    if arg == '--inherit-path':
+      args[i] = arg + '=prefer'
```

Comment:
potentially more readable/idiomatic to just craft a new list and replace the old args inline?

```
def transform_legacy_arg(arg):
  if arg == '--inherit-path':
    return '--inherit-path=prefer'
  return arg

args = args or sys.argv[1:]
args = [transform_legacy_arg(arg) for arg in args]
```

Reply:
Done